lib/screens/all_apps.dart

```diff
@@ -58,39 +58,95 @@ class AppsScreeen extends StatelessWidget {
   }
 }
 
-class AppWIdget extends StatelessWidget {
-  final apps = Get.put(MyApps());
+class AppWIdget extends StatefulWidget {
```

Comment:
lets not use stateful widget

Reply:
switch to stateless

Reply:
When you search for the apps the screen is changing state you need stateful widget!

Reply:
use oberbable make controller as explainedd below

Reply:
Sorry I haven't used `Observable` before.

Reply:
@ponder2000  take help from [here ](https://github.com/cimplesid/launcherx/blob/master/lib/controllers/apps.dart)